### PR TITLE
Fix missing "Install PWA" button for published empty Blazor WebAssembly app

### PR DESCRIPTION
**Pull request by cerasamson**
This PR fixes a bug in which the `Install PWA` button is missing from the browser in a published empty Blazor WebAssembly app, which was due to auth-related checks in `service-worker.published.js` transferred from the default template.

This bug was discovered by executing `dotnet BlazorApp-empty.Server.dll --urls "https://localhost:5001;http://localhost:5000"` after publishing the app to a folder.

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/wwwroot/service-worker.published.js [hunk 1]
```diff
@@ -42,13 +36,7 @@ async function onFetch(event) {
     let cachedResponse = null;
     if (event.request.method === 'GET') {
         // For all navigation requests, try to serve index.html from cache
-        // If you need some URLs to be server-rendered, edit the following check to exclude those URLs
-//#if(Hosted)
-        const shouldServeIndexHtml = event.request.mode === 'navigate'
-            && !event.request.url.includes('/connect/')
-//#else
         const shouldServeIndexHtml = event.request.mode === 'navigate';
-//#endif
```
Comment (TanayParikh) [inline src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/wwwroot/service-worker.published.js:51]: Is this related to authentication? Wondering why this may be necessary (possibly due to the `/connect/` endpoint?)

Comment (cerasamson) [reply src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/wwwroot/service-worker.published.js:51]: This was related to authentication, but this was modified to remove `IndividualLocalAuth` after being transferred from the default template. The `/connect/` endpoint is not really necessary in this case.

This is the original if statement in `/ComponentsWebAssembly-CSharp/Client/wwwroot/service-worker.published.js`:

```csharp
//#if(IndividualLocalAuth && Hosted)
        const shouldServeIndexHtml = event.request.mode === 'navigate'
            && !event.request.url.includes('/connect/')
            && !event.request.url.includes('/Identity/');

**Review by TanayParikh — APPROVED** [review]
LGTM, thanks for looking into this!
